getting-started.md

```diff
@@ -758,76 +758,73 @@ In this section, we will provide an overview of the supply chain security use ca
 
 ### Sign: Introducing Image Signing & Verification to your Supply Chain
 
-**Overview**
+#### Overview
+
+This component allows a platform operator to define a policy that will
+restrict unsigned images from running on clusters.
+To enforce the configured policies this component communicates with external
+container registries to verify signatures on container images and make a
+decision based on the results of this verification. In order to make admission
+decisions this component is implemented as a dynamic admission control webhook.
 
-This feature-set allows an application operator to define a policy that will restrict unsigned images from running on clusters.
-This is done using a dynamic admission control component on Kubernetes clusters.
-This component contains logic to communicate with external registries and verify signatures on container images,
-making a decision based on the results of this verification.
 Currently, this component supports cosign signatures and its key formats.
-It will work with open source cosign, kpack and Tanzu Build Service (which is what we will overview in this document).
+Although this component does not sign container images, you could use tools such
+as the cosign CLI, kpack, and Tanzu Build Service (which is what we will overview
+in this document) to generate signatures for your images.
 
-Signing an artifact creates metadata about it that allows consumers to verify its origin and integrity.
-By verifying signatures on artifacts prior to their deployment,
-this allows operators to increase their confidence that trusted software is running on their clusters.
+Signing an artifact creates metadata about it that allows consumers to verify
+its origin and integrity.
+Operators can increase their confidence that trusted software is running on their
+clusters by verifying signatures on artifacts prior to their deployment.
 
-**Use Cases**
+#### Use Cases
 
 * Validate signatures from a given registry.
+* Deny unsigned or unverified images from being admitted in the cluster.
```

Comment:
`unverified images`: that doesn't sound correct to me, when you deploy the webhook it doesn't deny the running pods, only when updating them or changing them. I'd simplify by mentioning `unsigned` images.